roles/rclone_serve_restic/defaults/container.yml

```diff
@@ -0,0 +1,64 @@
+---
```

Comment:
Ansible is doing a weird thing here. `defaults/container.yml` is not in scope. If you want to split among multiple files, they need to all be in `defaults/main/*.yml` instead. See the `*_from` options of https://docs.ansible.com/ansible/latest/collections/ansible/builtin/import_role_module.html and https://docs.ansible.com/ansible/latest/collections/ansible/builtin/include_role_module.html for some background, and https://github.com/famedly/ansible-collection-observability/tree/main/roles/victoriametrics/defaults/main for an example how we did it somewhere else